### PR TITLE
Added descriptions to commands.

### PR DESCRIPTION
**Pull request by soarn**
Modified `eval.rb, highlights.rb, join.rb, topic.rb, user.rb` to have accurate descriptions.

### CODE REVIEW

File: discord/eval.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module Eval
   extend Discordrb::Commands::CommandContainer
 
-  command(:eval) do |event, *code|
+  command(:eval, description: "Evals.") do |event, *code|
```
Comment (Chew) [inline discord/eval.rb]: Can be more descriptive

File: discord/highlights.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 module Highlights
   extend Discordrb::Commands::CommandContainer
 
-  command(:highlights, min_args: 1, max_args: 2) do |event, action, word|
+  command(:highlights, min_args: 1, max_args: 2, description: "Marks a phrase as alertable, and will alert you when that phrase is used.") do |event, action, word|
```
Comment (Chew) [inline discord/highlights.rb]: Can be modified beyond describing the add function

File: discord/join.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 module Join
   extend Discordrb::Commands::CommandContainer
 
-  command(:join, min_args: 0, max_args: 2) do |event, channel = nil, pass = nil|
+  command(:join, min_args: 0, max_args: 2, description: "Creates specifed channel in discord, and joins channel in IRC.") do |event, channel = nil, pass = nil|
```
Comment (Chew) [inline discord/join.rb]: Reverse the sentence, stuff before comma then what's after it

File: discord/topic.rb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 module Topic
   extend Discordrb::Commands::CommandContainer
 
-  command(:topic) do |event|
+  command(:topic, description: "Update's discord channel's topic to match that of the connected IRC channel.  Also displays channel modes and URL if one is set.") do |event|
```
Comment (Chew) [inline discord/topic.rb]: Doesn't display URL.

File: discord/highlights.rb (hunk 2 above)
Comment (soarn) [reply discord/highlights.rb]:
```suggestion
  command(:highlights, min_args: 1, max_args: 2, description: "Use `list` to view alertable phrases you have set up, use `add` and `remove` to add or remove alertable phrases respectively.") do |event, action, word|
```
I think this should be sufficient.  Does the [`] affect the actual ruby file?  If so what would you recommend me change it to?

File: discord/topic.rb [hunk 5]
```diff
@@ -1,7 +1,7 @@
 module Topic
   extend Discordrb::Commands::CommandContainer
 
-  command(:topic, description: "Update's discord channel's topic to match that of the connected IRC channel.  Also displays channel modes and URL if one is set.") do |event|
+  command(:topic, description: "UUpdate's discord channel's topic to match that of the connected IRC channel.  Also displays channel modes.") do |event|
```
Comment (Chew) [inline discord/topic.rb]: UUpdate

File: discord/highlights.rb (hunk 2 above)
Comment (Chew) [reply discord/highlights.rb]: how did you do that

File: discord/join.rb [hunk 6]
```diff
@@ -1,7 +1,7 @@
 module Join
   extend Discordrb::Commands::CommandContainer
 
-  command(:join, min_args: 0, max_args: 2, description: "Creates specifed channel in discord, and joins channel in IRC.") do |event, channel = nil, pass = nil|
+  command(:join, min_args: 0, max_args: 2, description: "Joins channel on IRC, and creates channel on Discord.") do |event, channel = nil, pass = nil|
```
Comment (Chew) [inline discord/join.rb]: creates the channel if it doesn't exist

File: discord/highlights.rb [hunk 7]
```diff
@@ -1,7 +1,7 @@
 module Highlights
   extend Discordrb::Commands::CommandContainer
 
-  command(:highlights, min_args: 1, max_args: 2, description: "Marks a phrase as alertable, and will alert you when that phrase is used.") do |event, action, word|
+  command(:highlights, min_args: 1, max_args: 2, description: "Use `list` to view alertable phrases you have set up, use `add` and `remove` to add or remove alertable phrases respectively.") do |event, action, word|
```
Comment (Chew) [inline discord/highlights.rb]: Maybe add "Manage your highlighted words." Keep the rest

File: discord/highlights.rb (hunk 2 above)
Comment (soarn) [reply discord/highlights.rb]: just hit the button in the commenting panel, it's the one with the **+** and **-** right before the header text.  (Or press `command + g`)

File: discord/eval.rb [hunk 8]
```diff
@@ -1,7 +1,7 @@
 module Eval
   extend Discordrb::Commands::CommandContainer
 
-  command(:eval, description: "Evals.") do |event, *code|
+  command(:eval, description: "Evals given input through the bot.") do |event, *code|
```
Comment (Chew) [inline discord/eval.rb]: Not at all what I told you to put

Comment (soarn) [reply discord/eval.rb]: How would you prefer I write it?  I don't use eval that much, so I'm glad to have your input.

Comment (Chew) [reply discord/eval.rb]: My mistake, I never specified. Put "Evaluates Ruby expressions." Then add something about "for Ruby experts, use with caution" or something

File: discord/topic.rb [hunk 9]
```diff
@@ -1,7 +1,7 @@
 module Topic
   extend Discordrb::Commands::CommandContainer
 
-  command(:topic) do |event|
+  command(:topic, description: "Update's discord channel's topic to match that of the connected IRC channel.  Also displays channel modes.") do |event|
```
Comment (Chew) [inline discord/topic.rb]:
```suggestion
  command(:topic, description: "Update's discord channel's topic to match that of the connected IRC channel. Also displays channel modes.") do |event|
```

Why's this space here

File: discord/join.rb [hunk 10]
```diff
@@ -1,7 +1,7 @@
 module Join
   extend Discordrb::Commands::CommandContainer
 
-  command(:join, min_args: 0, max_args: 2) do |event, channel = nil, pass = nil|
+  command(:join, min_args: 0, max_args: 2, description: "Joins channel on IRC, and creates channel on Discord if channel doesn't already exist.") do |event, channel = nil, pass = nil|
```
Comment (Chew) [inline discord/join.rb]:
```suggestion
  command(:join, min_args: 0, max_args: 2, description: "Joins a channel on IRC, and creates a channel on Discord if a channel doesn't already exist.") do |event, channel = nil, pass = nil|
```

File: discord/topic.rb [hunk 11]
```diff
@@ -1,7 +1,7 @@
 module Topic
   extend Discordrb::Commands::CommandContainer
 
-  command(:topic) do |event|
+  command(:topic, description: "Update's discord channel's topic to match that of the connected IRC channel. Also displays channel modes.") do |event|
```
Comment (Chew) [inline discord/topic.rb]:
```suggestion
  command(:topic, description: "Updates the current Discord channel's topic to match the connected IRC channel's. Also displays channel modes.") do |event|
```

**Pull request merged**